stack.yaml

```diff
@@ -1,11 +1,9 @@
-resolver: lts-9.0
+resolver: lts-9.6
 extra-deps:
   - configurator-ng-0.0.0.1
   - critbit-0.2.0.0
-  - hasql-pool-0.4.1
```

Comment:
Does the new LTS try including a newer version of hasql-pool that conflicted with the old choice of 0.4.1?

Reply:
The new LTS includes this [version](https://www.stackage.org/lts-9.6/package/hasql-pool-0.4.1) exactly